Projects/Features/SearchFeature/Sources/Reactors/SearchReactor.swift

```diff
@@ -57,14 +59,27 @@ final class SearchReactor: Reactor {
             newState.typingState = state
         case let .updateText(text):
             newState.text = text
+        case let .fetchRecentText(text):
+            break
         }
 
         return newState
     }
+
+    func transform(mutation: Observable<Mutation>) -> Observable<Mutation> {
+        let servicetypingState = DefaultSearchCommonService.shared.typingStatus
```

Comment:
```suggestion
        let servicetypingState = service.typingStatus
```
이러면 Service를 주입받은 의미가 없는거같아요

Reply:
앗 실수를 ..